integration-test/src/it/relocation-group/verify.groovy

```diff
@@ -46,3 +46,13 @@ assert profile1.size() != 0
 dependency = pom.dependencyManagement.dependencies.dependency.find { it.artifactId.text() == "junit-dep" }
 assert dependency != null
 assert dependency.version.text() == "4.1"
+
+
+def passed = false
+pom.properties.each {
+    if ( it.text().contains ("slf4j") )
+    {
+        passed = true
```

Comment:
This can't break, so once true, always true. Maybe we just want a `for` loop instead? Or `find`? Several ways to do it.

Reply:
Indeed ; it was a copy and paste from how other tests did it.

Reply:
Given that there are so many of this pattern I'm think it can be left as is ; any changes should be done en-mass in a separate PR IMHO.